mantidproject/mantid
# Separate models in indirect data analysis - IndirectFitDataPresenter

**Description of work.**

This PR separates aspects of the IndirectFitDataPresenter from the IndirectFittingModel and merges it with IndirectFitDataTable in to it's own model.

**To test:**

There should be no changes to fucntionality with this PR. It is important to check that the standard usage of all the tabs opperate correctly.

Partial #29210

*This does not require release notes* because **It is entirly internal changes**

Finally, don't forget to add the appropriate labels, milestones, etc.!  -->

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
